django/test/runner.py

```diff
@@ -97,25 +97,51 @@ def debug(self, error):
         pdb.post_mortem(traceback)
 
 
-class RemoteTestResult:
+class DummyList:
     """
-    Record information about which tests have succeeded and which have failed.
+    Dummy list class for faking storage of results in unittest.TestResult. We
+    do this because RemoteTestResult replaces the way of recording.
+    """
+    __slots__ = []
 
-    The sole purpose of this class is to record events in the child processes
-    so they can be replayed in the master process. As a consequence it doesn't
-    inherit unittest.TestResult and doesn't attempt to implement all its API.
+    def append(self, item):
+        pass
 
-    The implementation matches the unpythonic coding style of unittest2.
+
+class RemoteTestResult(unittest.TestResult):
```

Comment:
Can we move subclassing `unittest.TestResult` to a separate commit? Is it a valuable optimization on its own? or it's strictly related with supporting `--buffer` with `--parallel`.

Reply:
Done